### PR TITLE
Add generate_wombatlite_ic.py script

### PR DESCRIPTION
**Pull request by dougiesquire**
This PR adds a simple script for generating the ACCESS-OM3 WOMBATlite initial conditions for those tracers that are initalised to constants (a la ACCESS-OM2).

### CODE REVIEW

File: wombat_ic_generation/generate_wombatlite_ic.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Copyright 2023 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# =========================================================================================
+# Generate constant initial conditions for WOMBATlite using grid from a provided file
+#
+# To run:
+#   python generate_wombatlite_ic.py --grid-filename=<path-to-grid-file>
+#       --variable-name=<variable-name> --ic-filename=<path-to-output-file>
+#
```
Comment (anton-seaice) [inline wombat_ic_generation/generate_wombatlite_ic.py:10]: Its not clear to me from this comment what type of grid file is expected, or what the variable name is for ? (Can there be a default variable name ?)

**Review by anton-seaice — COMMENTED** [review]
I think if this had an example use it would be clearer what grid file was needed etc . 

In /g/data/vk83/configurations/inputs/access-om3/wombat/initial_conditions/global.1deg/2024.04.02/init_ocean_wombatlite.res.nc, there is a time dimension. Is that needed ? 

It looks like this needs ocean initial conditions as a template? If that is true, that should be clearer in the description 

Also, cchecker.py shows many things but maybe these are worth setting:

```
variable "caco3" missing the following attributes:
* coverage_content_type
* standard_name

variable "caco3_sediment" missing the following attributes:
* coverage_content_type
* standard_name

variable "det" missing the following attributes:
* coverage_content_type
* standard_name

variable "det_sediment" missing the following attributes:
* coverage_content_type
* standard_name

variable "phy" missing the following attributes:
* coverage_content_type
* standard_name

variable "time" missing the following attributes:
* standard_name

variable "zoo" missing the following attributes:
* coverage_content_type
* standard_name
*```

File: wombat_ic_generation/generate_wombatlite_ic.py (hunk 1 above)
Comment (dougiesquire) [reply wombat_ic_generation/generate_wombatlite_ic.py:10]: This is described somewhat in the help output:

```bash
$ python generate_wombatlite_ic.py -h
usage: generate_wombatlite_ic.py [-h] --grid-filename GRID_FILENAME --variable-name VARIABLE_NAME --ic-filename
                                 IC_FILENAME

Generate constant initial conditions for WOMBATlite using grid from provided file.

options:
  -h, --help            show this help message and exit
  --grid-filename GRID_FILENAME
                        The path to a file containing a variable to copy the grid from.
  --variable-name VARIABLE_NAME
                        The name of the variable to copy the grid from.
  --ic-filename IC_FILENAME
                        The path to the initial condition file to be outputted.
```

**Comment by dougiesquire** [comment]
> Also, cchecker.py shows many things but maybe these are worth setting:
>
> ```
> variable "caco3" missing the following attributes:
> * coverage_content_type
> * standard_name
> ```

What is `coverage_content_type`? I don't think many of these variables have valid CF standard names.

File: wombat_ic_generation/generate_wombatlite_ic.py (hunk 1 above)
Comment (dougiesquire) [reply wombat_ic_generation/generate_wombatlite_ic.py:12]: Resolved in [34b6caa](https://github.com/COSIMA/om3-scripts/pull/23/commits/34b6caa9d19840aa2c41720b93010db2b685dd75)

File: wombat_ic_generation/generate_wombatlite_ic.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# Contact:
+#   Dougie Squire <dougie.squire@anu.edu.au>
+#
+# Dependencies:
+#   argparse, xarray
+# =========================================================================================
+
+import os
+import subprocess
```
Comment (anton-seaice) [inline wombat_ic_generation/generate_wombatlite_ic.py:29]:
```suggestion
```

**Review by anton-seaice — APPROVED** [review]
Looks good :)

**Pull request merged**